### ISSUE TITLE
ARIA script deprecating 4 attributes needs to somehow include DPUB-ARIA subclasses

### ISSUE THREAD

**Issue by carmacleod** [w3c/dpub-aria#31]
@jnurthen dpub-aria roles that inherit from section or landmark (maybe others) are missing "(deprecated on this role in ARIA 1.2)" on `aria-disabled`, `aria-errormessage`, `aria-haspopup`, and `aria-invalid`. Do you know what needs to be done to fix this?

**Comment by jnurthen** [comment]
I was not aware how this works but it seems like roleInfo.js needs updating and then updates need to be put into ariaChild.js
